go/teams/loader.go

```diff
@@ -909,9 +907,15 @@ func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (
 	}
 
 	if hd := hiddenPackage.ChainData(); hd != nil {
+		hd.NeedRotate = needHiddenRotate
 		load2res.hidden = hd
 	}
 
+	if !arg.fromBoxAudit && needHiddenRotate {
+		mctx.Info("Scheduling box audit since we needed a rotation upon load (and we're not being directly called from the box audit)")
+		go BoxAuditAfterBackoff(mctx, arg.teamID)
```

Comment:
@heronhaye what happens if a bunch of these pile up. Will the BoxAuditor internally discard the later ones?